commands/fetchmitre.go

```diff
@@ -0,0 +1,62 @@
+package commands
+
+import (
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	db "github.com/vulsio/go-cve-dictionary/db"
+	log "github.com/vulsio/go-cve-dictionary/log"
```

Comment:
These aliases are not needed?

Reply:
https://github.com/vulsio/go-cve-dictionary/pull/392/commits/46146adab394c9315c12f78ee4d22d3380831beb